tools/run_tests.sh

```diff
@@ -28,6 +28,7 @@ generic_client_path="${TCF_HOME}/examples/apps/generic_client"
 # Read Listener port from config file
 listener_port=`grep listener_port ${TCF_HOME}/config/tcs_config.toml | awk {'print $3'}`
 LISTENER_URL="localhost"
+WORKER_ID="singleton-worker-1"
```

Comment:
Please add a comment that this value should match with singleton enclave manager worker id

Reply:
Done